pages/my-account.vue

```diff
@@ -92,19 +147,120 @@ export default defineComponent({
         },
       ],
       orders: [
-        ["#45", "23th June, 2021", "Visa card", "$412.00", "Finalised"],
-        ["#46", "26th June, 2021", "Paypal", "$132.00", "Finalised"],
-        ["#47", "28th June, 2021", "Mastercard", "$12.00", "Finalised"],
-        ["#48", "28th June, 2021", "Paypal", "$20.00", "In process"],
+        {
+          orderNumber: "#45",
+          submittedDate: "23th June, 2021",
+          products: "Men's Performance Socks,Women's Teedsasdfa",
+          total: "$412.00",
+          status: "Finalised",
+        },
+        {
+          orderNumber: "#46",
+          submittedDate: "24th June, 2021",
+          products: "AAAAABB",
+          total: "$402.00",
+          status: "Processing",
+        },
+        {
+          orderNumber: "#47",
+          submittedDate: "25th June, 2021",
+          products: "CCCFFF SDD",
+          total: "$512.00",
+          status: "Finalised",
+        },
+        {
+          orderNumber: "#48",
+          submittedDate: "28th June, 2021",
+          products: "RRRRRRRRRRRR GGGGGG",
+          total: "$612.00",
+          status: "Delivered",
+        },
+      ],
+      addresses: [
+        {
+          // @TODO hardecoded need to be removed
+          accountId: 1366,
+          types: [
+            {
+              name: "Shipping",
+              isPrimary: true,
+            },
+            {
+              name: "Billing",
+              isPrimary: true,
+            },
+          ],
+          auditInfo: {
+            updateDate: 1638834415766,
+            createDate: 1638834415766,
+            updateBy: "tbd",
+            createBy: "tbd",
+          },
+          faxNumber: null,
+          label: null,
+          id: 1243,
+          email: "kevin.watts@kibocommerce.com",
+          firstName: "kevin",
+          middleNameOrInitial: null,
+          lastNameOrSurname: "watts",
+          companyOrOrganization: null,
+          phoneNumbers: {
+            home: "1231231234",
+            mobile: null,
+            work: null,
+          },
+          address: {
+            address1: "2717 south lamar",
+            address2: "b247",
+            address3: null,
+            address4: null,
+            cityOrTown: "austin",
+            stateOrProvince: "TX",
+            postalOrZipCode: "78704",
+            countryCode: "US",
+            addressType: "Residential",
+            isValidated: true,
+          },
+        },
       ],
     })
 
+    const goBack = () => {
+      if (steps.value.length) {
+        barTitle.value = steps.value.pop()
+        activePage.value = barTitle.value
+      } else {
+        context.root.$router.go(-1)
+      }
+    }
+    const changeActivePage = async (title) => {
+      if (title === "Log out") {
+        await logout()
+        app.router.push({ path: "/" })
+        return false
+      }
+    }
+
+    const gotoOrderHistory = (title) => {
+      steps.value.push(barTitle.value)
```

Comment:
shouldn't we use an actual route for the order history page?
app.router.push({ path: "/my-account/order-history" } )

Reply:
Yes we can, implemented the same. I assumed to put in one route only.